ui/component/core/src/util.ts

```diff
@@ -1056,3 +1056,18 @@ export function enableScroll() {
     window.removeEventListener('touchmove', preventDefault, wheelOpt);
     window.removeEventListener('keydown', preventDefaultForScrollKeys, false);
 }
+
+export function convertBase64 (file:any) {
+    return new Promise((resolve, reject) => {
+        const fileReader = new FileReader();
+        fileReader.readAsDataURL(file);
+
+        fileReader.onload = () => {
+            resolve(fileReader.result);
+        };
+
+        fileReader.onerror = (error) => {
+            reject(error);
+        };
+    });
+}
```

Comment:
* `file:any` is not true...`FileReader` expects a `Blob` object 
* This should be named `blobToBase64(blob:Blob)` and also it should used the code from `or-asset-viewer/index.ts:469` (where the URL prefix is stripped) and then this util method can be re-used here and in `or-asset-viewer`